recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "trulens-core" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trulens_core-{{ version }}.tar.gz
+  sha256: bfbadd53a5a5d17575a8855f3fd888cad47912eb53fc4b0f27487ad341ba9971
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 100
+  skip: true  # [py<39]
+
+requirements:
+  host:
+    - python
+    - poetry-core
+    - pip
+  run:
+    - python
+    - numpy >=1.23
+    - munch >=2.5.0,<3.0.0
+    - dill >=0.3.8
+    - typing_extensions >=4.9.0,<5.0.0
+    - pydantic >=2.0.0,<3.0.0
+    - pandas >=2.1.0,<3.0.0
```

```suggestion
    - pandas >=1.0.0
```
It seems the only difference in deps is relaxing the restraint on pandas [here](https://github.com/truera/trulens/blob/trulens-1.1.0/src/core/pyproject.toml#L36).